src/View/UIManager.java

```diff
@@ -108,6 +108,8 @@ public void mainMenu() throws Exception
         // Set the scene:
         mainStage.setScene(new Scene(root, 1000, 750, true, SceneAntialiasing.BALANCED));
         mainStage.setTitle("PearPlanner");
+        mainStage.setMinWidth(1000);
```

Comment:
Does it make sense to set the minimum height/width like this?  That is, on a small screen/device this might make the interface unusable.  Perhaps the minimum sizes should be based on the display size/resolution.

Reply:
I completely understand what you mean. I had to switch from a FlowPane to a GridPane to allow for responsiveness the only issue with that is now the panes cannot wrap like before. So if I set the minimum screen width to something smaller the last two items will be cut off.  Since we haven't built a mobile version yet I didn't know if that was necessary right away of I could continue to work on issues like this later one with each new implementation.

Reply:
OK.  That makes sense.  I am OK with it staying like for this for now, but you should a comment that more or less explains what you just explained about the current limitation how it will be addressed in the future.